software-containers/README.md

```diff
@@ -16,3 +16,18 @@ group:      sss files
 
 ### xterm
 `xterm` must be installed on the container
+
+### ENTRYPOINT
+Use ENTRYPOINT to specify an executable to be run, for example a startup script which can execute several tasks when the container starts up.
+
+### CMD
+Use CMD to specify a command to be executed at the end of the Dockerfile, "xterm".
+
+### desktop
+Each container is associated with a desktop file, for example terminal.desktop, which includes the path to the shell script, for example terminal.sh, which initializes and starts up the container.
+
+### shell script
+A shell script is created for each container. The shell script initializes and starts up the container. As a result, PostAction is invoked. PostAction uses a default parameter to start up an xterm. However the shell script can pass a parameter to PostAction, which replaces the default parameter. The following is an example script that passes a parameter to PostAction:
+```
```

Comment:
A shell script isn't required for a container to run in arcade.  It is a good option to document though.

"A shell script is created" makes it sound like it is automated somehow.  It would be the users creating such a script.

Reply:
Yes, the curl command can be executed manually.

Creation of a shell script is automated in ARCADE via the create-software-links.sh invoked within Makefile, no? Although if we want to port this README.md file to another project, this may not be true.